packages/terra-alert/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Update docs that were missed during mdx upgrade
```

Comment:
Can you update this changelog to reflect the changes made for terra-alert?

Reply:
Updated changelog for all packages here https://github.com/cerner/terra-core/pull/2923/commits/f87b07d33395abb007223d290ab782cc24bba149.